en/V1-IoT_Ecosystem_Requirements.md

```diff
@@ -29,7 +29,7 @@ To ensure the security of all software produced, the build process for the syste
 | -- | ---------------------- | - | - | - |
 | **1.2.1** | Verify that each application (including firmware) in the ecosystem maintains a software bill of materials (SBOM) cataloging third-party components, versioning, and published vulnerabilities. | ✓ | ✓ | ✓ |
 | **1.2.2** | Verify that potential areas of risk that come with the use of third-party and open-source software have been identified and that actions to mitigate such risks have been taken. | ✓ | ✓ | ✓ |
-| **1.2.3** | Verify the device is released with firmware and configuration appropriate for a release build (as opposed to debug versions). | ✓ | ✓ | ✓ |
+| **1.2.3** | Verify the device is released with firmware configured with secure defaults appropriate for a release build (as opposed to debug versions). | ✓ | ✓ | ✓ |
```

Comment:
Thanks for the PR here. We are mixing Chapter 1 and chapter 3 commits but it's not a big deal.

Reply:
Definitely not what I was trying to do oops 😞 
I'm going to update it and push a new PR.
I am a huge noob in GitHub and I don't know how to remove some files from the PR.